lib/symboldatabase.cpp

```diff
@@ -3550,6 +3550,17 @@ static const Token* skipPointers(const Token* tok)
     return tok;
 }
 
+static const Token* skipPointersAndQualifiers(const Token* tok)
```

Comment:
Perhaps this should be merged with ```skipPointers()``` (in that case, the ```Match()```-expression would just have to be extended with ```const|volatile``` and we would be done, but I wasn't sure that would be ok from the other place ```skipPointers()``` was called?